init-tools.cmd

```diff
@@ -77,6 +77,9 @@ if %UPDATE_CLI_ERRORLEVEL% GTR 1 (
   exit /b %UPDATE_CLI_ERRORLEVEL%
 )
 
+echo Copying supplemental overrides from Tools-Override.
```

Comment:
More hacky stuff so we can quickly update things in buildtools without going through that process.